test/e2e/test/elasticsearch/steps_mutation.go

```diff
@@ -155,10 +155,13 @@ func (b Builder) MutationTestSteps(k *test.K8sClient) test.StepList {
 				},
 				Test: func(t *testing.T) {
 					continuousHealthChecks.Stop()
+
 					for _, f := range continuousHealthChecks.Failures {
 						t.Errorf("Elasticsearch cluster health check failure at %s: %s", f.timestamp, f.err.Error())
 					}
-					assert.Equal(t, 0, continuousHealthChecks.FailureCount)
+					if !b.relaxMutationHealthChecks {
+						assert.Equal(t, 0, continuousHealthChecks.FailureCount)
+					}
```

Comment:
Primarily just a question. "relaxing" the mutation health checks to me seems like you'd allow a "couple" of failures, but this is allowing 1000s of failures. What are your thoughts on simply adjusting the max failures to be something like 10 or 5 when we're relaxing them? (I'm unsure how many we're seeing in these scenarios that we're trying to prevent.

Reply:
I started with a counter, then I wasn't sure what value to use, so I switched to a boolean. But yes, said like that, a counter seems safer.
We often see 9 or 10 failed health checks.